learning_wavelets/models/learnlet_layers.py

```diff
@@ -139,18 +139,32 @@ def __init__(
         if self.tiling_unit_norm:
             constraint = UnitNorm(axis=[0, 1, 2])
         tiling_prefix = 'details_tiling'
-        self.convs_detail_tiling = [
+        self.convs_detail_tiling_fixed = [
             Conv2D(
-                n_tiling,
+                n_tiling//2,
                 self.kernel_size,
                 activation='linear',
                 padding='same',
                 kernel_initializer='glorot_uniform',
                 use_bias=tiling_use_bias,
                 kernel_constraint=constraint,
+                trainable = False,
                 name=f'{tiling_prefix}_{str(K.get_uid(tiling_prefix))}',
```

Comment:
```suggestion
                name=f'{tiling_prefix}_fixed_{str(K.get_uid(tiling_prefix))}',
```

Reply:
@kevinmicha I don't understand why you resolved this

Reply:
was a mistake. Now really solved